symbolic-debuginfo/src/elf.rs

```diff
@@ -4,10 +4,11 @@ use std::borrow::Cow;
 use std::fmt;
 use std::io::Cursor;
 
-use failure::Fail;
 use flate2::{Decompress, FlushDecompress};
 use goblin::elf::compression_header::{CompressionHeader, ELFCOMPRESS_ZLIB};
-use goblin::{container::Ctx, elf, error::Error as GoblinError, strtab};
+pub use goblin::error::Error as GoblinError;
```

Comment:
_nit_: Lets move this under the imports as this is now a re-export. Either inherit docs or add a doc string.